Core/HLE/proAdhocServer.cpp

```diff
@@ -2075,7 +2075,7 @@ int server_loop(int server)
 		sleep_ms(1);
```

Comment:
This should really be a `select()` and there should be no sleeps.

In case it's helpful, see this code that does a bunch of non-blocking socket operations:
https://github.com/hrydgard/ppsspp/blob/master/ext/native/net/websocket_server.cpp

-[Unknown]

Reply:
it was the old code from aemu project, i didn't touch the logic much

actually, `select` with timeout will also sleeping under the hood, since it's blocking the thread :)

Reply:
Right, but performance should be better with `select()` since it'll act on new network data much quicker.

-[Unknown]